src/main/kotlin/com/grepp/nbe1_3_team04/vote/domain/VoteItemDate.kt

```diff
@@ -0,0 +1,29 @@
+package com.grepp.nbe1_3_team04.vote.domain
+
+import jakarta.persistence.Entity
+import java.time.LocalDateTime
+
+@Entity
+class VoteItemDate private constructor(
+    vote: Vote,
+    time: LocalDateTime
+) : VoteItem(vote) {
+
+    val time: LocalDateTime
+
+    init {
+        vote.endAt?.let { validateTime(time, it) }
+        this.time = time
+    }
```

Comment:
해당 부분은 컨벤션대로면 
```suggestion
    val time: LocalDateTime = time

    init {
        vote.endAt?.let { validateTime(time, it) }
    }
```
이렇게 변경해주시면 됩니다!

Reply:
감사합니다 ~ 수정하겠습니다!